packages/chart/src/components/BarChart.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useContext, useState, useEffect } from 'react'
+import React, { useContext, useState, useEffect, useDebugValue } from 'react'
```

Comment:
Is `useDebugValue` used anywhere? I'm wondering if we should remove it. Either way, I'm happy to see you used it while working through the ticket. I've never used it, so I was able to learn something new 🙂

Reply:
I can remove it if you want me to.  Sorry forgot that.

Reply:
Yeah go ahead and remove it, I'm happy it was there for me to learn :)